warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__feed_guideline_index.sql

```diff
@@ -0,0 +1,33 @@
+{{ config(materialized='table') }}
+
+WITH gtfs_schedule_fact_daily_feeds AS (
+    SELECT * FROM {{ ref('gtfs_schedule_fact_daily_feeds') }}
+),
+
+gtfs_schedule_dim_feeds AS (
+    SELECT * FROM {{ ref('gtfs_schedule_dim_feeds') }}
+),
+
+-- list all the checks that have been implemented
+checks_implemented AS (
+    SELECT {{ static_feed_downloaded_successfully() }} AS check, {{ compliant_on_the_map() }} AS feature
+    UNION ALL SELECT {{ no_validation_errors_in_last_30_days() }}, {{ compliant_on_the_map() }}
```

Comment:
nit: formatting

also do these need to be macros/variables anyways?

Reply:
I had them as macros/variables because we will use them in every table and they're just hard-coded strings.... Want it to be able to just tweak those in one place

Reply:
https://github.com/cal-itp/data-infra/pull/1712/commits/6ba71b6bec8fa5a6761e110e8395f11e17d858f0